charts/rasa-x/templates/event-service-deployment.yaml

```diff
@@ -57,10 +57,12 @@ spec:
           httpGet:
             path: "/health"
             port: "http"
+            scheme: {{ default "HTTP" .Values.eventService.probeScheme }}
         readinessProbe:
           httpGet:
             path: "/health"
             port: "http"
+            scheme: {{ default "HTTP" .Values.eventService.probeScheme }}
```

Comment:
I think I could make a distinction between probes, e.g.

```
readinessProbe:
  scheme: {{ default "HTTP" .Values.eventService.readinessProbe.scheme }}
livenessProbe:
  scheme: {{ default "HTTP" .Values.eventService.livenessProbe.scheme }}
```

And the same for the rest of the deployments.

Reply:
Sure thing, makes sense!